src/scss/custom.scss

```diff
@@ -19,15 +19,14 @@
   --ifm-hover-overlay: rgba(93, 52, 242, 0.12);
   --ifm-code-font-size: 95%;
   --ifm-navbar-height: 64px;
-  --doc-sidebar-width:340px !important;
+  --doc-sidebar-width: 340px !important;
```

Comment:
Gao has changed this to 380px. Conflict merging required

Reply:
k

Reply:
rebased